apps/src/templates/teacherDashboard/TeacherDashboardNavigation.jsx

```diff
@@ -46,193 +41,41 @@ const teacherDashboardLinks = [
-  };
-
-  handleScroll = () => {
-    const {listPosition} = this.state;
-    const navbarRef = this.refs.navbar;
-    const scrollLeft = $(navbarRef).scrollLeft();
-    const maxScrollLeft =
-      $(navbarRef)[0].scrollWidth - $(navbarRef)[0].clientWidth;
-    const inMiddle = scrollLeft > 0 && scrollLeft < maxScrollLeft;
-
-    // Update listPosition in state if necessary
-    if (inMiddle && listPosition !== ListPosition.middle) {
-      this.setState({listPosition: ListPosition.middle});
-    } else if (scrollLeft === 0 && listPosition !== ListPosition.start) {
-      this.setState({listPosition: ListPosition.start});
-    } else if (
-      scrollLeft === maxScrollLeft &&
-      listPosition !== ListPosition.end
-    ) {
-      this.setState({listPosition: ListPosition.end});
-    }
-  };
-
-  render() {
-    const {listPosition, shouldScroll} = this.state;
-    const links = this.state.linksToShow;
-    const containerStyles = this.state.shouldScroll
-      ? {...styles.container, ...styles.scrollableContainer}
-      : styles.container;
-    const chevronStyles = userAgentParser.isSafari()
-      ? {...styles.chevron, ...styles.safariSticky}
-      : styles.chevron;
-
-    return (
-      <div
-        ref="navbar"
-        id="uitest-teacher-dashboard-nav"
-        style={containerStyles}
-        onScroll={this.handleScroll}
-      >
-        {listPosition !== ListPosition.start && shouldScroll && (
-          <FontAwesome
-            icon="chevron-left"
-            style={{...chevronStyles, ...styles.chevronLeft}}
-            onClick={() => this.scrollTo(ListPosition.start)}
-          />
-        )}
-        {links.map(link => (
-          <NavLink
-            key={link.url}
-            to={link.url}
-            style={styles.linkContainer}
-            activeStyle={styles.activeLinkContainer}
-          >
-            <div style={styles.link}>{link.label}</div>
-          </NavLink>
-        ))}
-        {listPosition !== ListPosition.end && shouldScroll && (
-          <FontAwesome
-            icon="chevron-right"
-            style={{...chevronStyles, ...styles.chevronRight}}
-            onClick={() => this.scrollTo(ListPosition.end)}
-          />
-        )}
-      </div>
-    );
-  }
+export default function TeacherDashboardNavigation({links, showAITutorTab}) {
+  const aiTutorLinks = showAITutorTab
+    ? [
+        {
+          label: i18n.aiTutor(),
+          url: TeacherDashboardPath.aiTutorChatMessages,
+        },
+      ]
+    : [];
+  const renderedLinks = [...(links || teacherDashboardLinks), ...aiTutorLinks];
+
+  return (
+    <div id="uitest-teacher-dashboard-nav" className={styles.navContainer}>
+      {renderedLinks.map(link => (
+        <NavLink
+          key={link.url}
+          to={link.url}
+          className={styles.linkContainer}
+          activeClassName={styles.activeLinkContainer}
+        >
+          <div className={styles.link}>{link.label}</div>
```

Comment:
I think these are pretty important places to add aria-roles and aria-labels

Reply:
Reading up on `aria-label`s, I see this note in the [MDN page about them](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

> Note: aria-label is intended for use on interactive elements, or elements made to be interactive via other ARIA declarations, when there is no appropriate text visible in the DOM that could be referenced as a label

These links have clear text labels, so it seems the advice here is to not use them in this case. What do you think?

As for the labels, the component renders the link text inside a `div` nested under an `a` tag, and `a` tags have [the `link` role by default](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/link_role). Does the nested `div` create a need to explicitly tag it with `aria-role="link"` again?